src/model/serialize_ctx.rs

```diff
@@ -1,25 +1,24 @@
+use std::collections::HashMap;
+
+use chrono::{DateTime, Utc};
+use itertools::Itertools;
+use serde::Serialize;
 use wasm_bindgen::JsValue;
 
+use stremio_core::deep_links::SearchHistoryItemDeepLinks;
 use stremio_core::models::ctx::Ctx;
+use stremio_core::types::{
+    events::Events, notifications::NotificationItem, profile::Profile, resource::MetaItemId,
+};
+
+use crate::model::deep_links_ext::DeepLinksExt;
 
 pub fn serialize_ctx(ctx: &Ctx) -> JsValue {
     JsValue::from_serde(&model::Ctx::from(ctx)).unwrap()
 }
 
 mod model {
-    use std::collections::HashMap;
```

Comment:
Is there a reason this was moved to the super mod? These imports are not used there but only in `model` and they are not publicly re-exported. These should remain as they were.

Reply:
The reason was to follow how every other files declared imports